cs_api_cli/cs_api_cli.ml

```diff
@@ -62,7 +62,7 @@ let upload_trace
                (Cs_api_core.build_file_upload_request ~s3_url ~s3_signature
                   ~file)))
   >>= Cs_api_io.send_request ~client
-  >>= Cs_api_io.get_response
+  >>= Cs_api_io.get_response_parse_error
```

Comment:
Does the use of `get_response_parse_error` make sense here? The S3 API doesn't use GraphQL so I don't understand how `get_response_parse_error` (which is GraphQL-specific, as far as I can tell) would work here.

Reply:
That was the reason why `get_response` was still here at first. It didn't really make sense but worked because `Yojson.Safe.from_string body` failed so `get_response_parse_errors` worked like the old `get_response`.

I'll put `get_response` back.